### DISCUSSION TITLE
Can gapseq work for strains?

### DISCUSSION THREAD

**Discussion by cdiener** [jotech/gapseq#129] [Q&A]
Hi, we had good results using gapseq for species models, but get fairly bad results with the default workflows for strain models from the same species. Basically, gapseq will generate the same model for each strain even in the presence of significant inter-strain heterogeneity and experimentally observed heterogeneous phenotypes and growth rates in the same medium. Are there maybe some parameters one can set to improve this?

**Comment by jotech** [comment]
hi @cdiener 
thanks for bringing up this interesting issue! Is the strain variability still present at the pathway level? Could you maybe provide an example so that I can investigate a bit?

**Reply by Waschina** [reply]
good and valid question...

In addition to the observed phenotypic heterogeneity, do these strains exhibit known genotypic heterogeneity with respect to genes with metabolic functions?

Maybe we can look into the reactions that were added during gapfilling and compare them between strains with different growth phenotypes?

**Reply by cdiener** [reply]
Sorry, replied and not as a reply here. This particular dataset can't be shared for now, but we have a lot of strain genomes from BIOML, so I'll try to build those and will be able to share the FASTAs with you.

**Reply by jotech** [reply]
this would be nice, please let us know how it works!

**Comment by cdiener** [comment]
Yep, lots of differences in the genotype. CARVEME for instance does build pretty different models for the strains though they don't always match the phenotypes/co-cultures that well.
